src/pages/ReimbursementAccount/CompanyStep.js

```diff
@@ -176,7 +176,7 @@ class CompanyStep extends React.Component {
         }
 
         const incorporationDate = moment(this.state.incorporationDate).format(CONST.DATE.MOMENT_FORMAT_STRING);
-        BankAccounts.setupWithdrawalAccount({...this.state, incorporationDate});
+        BankAccounts.setupWithdrawalAccount({...this.state, incorporationDate, companyTaxID: this.state.companyTaxID.replace(/\D/g, '')});
```

Comment:
NAB: @eVoloshchak we could move this regex to const too

Reply:
True, updated the PR